src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -39,7 +40,9 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 			};
 		});
 
-		await this.collection.bulkWrite(operations);
+		const result: BulkWriteOpResultObject = await this.collection.bulkWrite(operations);
```

Comment:
Is there a deleted doc count?

Reply:
There is one. However, this update operation does not delete documents. It looks at each document in the collection view and performs an update _if that document is already present in the collection_ (notice the `upsert` beign false). 
The end result is that there are no deletions. I didn't see the point of consistently printing `Deleted 0 documents`.